client/app/pages/queries/components/QueryPageHeader.jsx

```diff
@@ -68,8 +69,8 @@ export default function QueryPageHeader({
   headerExtra,
   tagsExtra,
   onChange,
-  onRefresh,
 }) {
+  const isMobile = !useMedia({ minWidth: 768 });
```

Comment:
```suggestion
  const isMobile = useMedia({ maxWidth: 768 });
```

Reply:
That one was actually to make sure it had the same media rule as the CSS has (even though it should differ in only 1px :sweat_smile:)

Reply:
How about 
```js
const isDesktop = useMedia({ minWidth: 768 });
```
?